test/libsolidity/util/TestFileParser.cpp

```diff
@@ -107,6 +107,12 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 						if (lowLevelCall)
 							call.kind = FunctionCall::Kind::LowLevel;
 
+						if (m_builtins && m_builtins->count(call.signature) > 0)
+						{
+							checkBuiltinFunction(call.signature);
```

Comment:
We already looked for the signature, didn't we? So this function should do nothing.

Reply:
Yes, then just removing the call will be fine. Or other option would be to refactor `checkBuiltinFunction` to not throw but instead return `bool` saying if signature is builtin or not...
Than use it like
```
if (checkBuiltinFunction(call.signature))
	call.kind = FunctionCall::Kind::Builtin;
```